packages/@aws-cdk/aws-applicationsignals-alpha/lib/slo/metric.ts

```diff
@@ -0,0 +1,357 @@
+     * Used to reference this query in math expressions
+     * Must be unique within the set of queries
+     *
+     * @required
+     */
+    readonly id: string;
+
+    /**
+     * AWS account ID where the metric is located
+     *
+     * @default - current account
+     */
+    readonly accountId?: string;
+
+    /**
+     * The math expression
+     * Cannot be specified if metricStat is specified
+     *
+     * @default - undefined
+     */
+    readonly expression?: string;
+
+    /**
+     * The metric statistic configuration
+     * Cannot be specified if expression is specified
+     *
+     * @default - undefined
+     */
+    readonly metricStat?: MetricStat;
+
+    /**
+     * Whether this query should return data to be used in results
+     * Set to true for queries that produce final values
+     * Set to false for intermediate calculations
+     *
+     * @default false
+     */
+    readonly returnData?: boolean;
+}
+
+/**
+ * Period-based metric properties with Application Signals
+ */
+export interface PeriodBasedAppSignalsMetricProps extends SliMetricBaseProps {
+    /**
+     * The type of metric being measured
+     * Can be LATENCY or AVAILABILITY
+     *
+     * @required
+     */
+    readonly metricType: MetricType;
+
+    /**
+     * Key attributes for the service being monitored
+     * Must include at least one of Type, Name, and Environment
+     *
+     * @required
+     */
+    readonly keyAttributes: { [key: string]: string };
+
+    /**
+     * The name of the operation being measured
+     * Used to filter metrics for specific operation
+     *
+     */
+    readonly operationName?: string;
+
+    /**
+     * The period in seconds for metric aggregation
+     * Must be a multiple of 60
+     *
+     * @required
+     */
+    readonly periodSeconds: number;
+
+    /**
+     * The statistic to use for aggregation
+     * Examples: Average, Sum, p99
+     *
+     * @required
+     */
+    readonly statistic: MetricStatistic;
+}
+
+/**
+ * Period-based metric properties with CloudWatch metrics
+ */
+export interface PeriodBasedCloudWatchMetricProps extends SliMetricBaseProps {
+    /**
+     * The metric data queries to execute
+     * Can include raw metrics and math expressions
+     *
+     * @required
+     */
+    readonly metricDataQueries: MetricDataQuery[];
+
+    /**
+     * The period in seconds for metric aggregation
+     * Must be a multiple of 60
+     *
+     * @required
+     */
+    readonly periodSeconds: number;
+
+    /**
+     * The statistic to use for aggregation
+     * Examples: Average, Sum, p99
+     *
+     * @required
+     */
+    readonly statistic: MetricStatistic;
+}
+
+/**
+ * Request-based metric properties with Application Signals
+ */
+export interface RequestBasedAppSignalsMetricProps extends SliMetricBaseProps {
+    /**
+     * The type of metric being measured
+     * Can be LATENCY or AVAILABILITY
+     *
+     * @required
+     */
+    readonly metricType: MetricType;
```

Comment:
I can see below that you defined some default value for it, so please add the default value here in JS Docs using `@default` annotation

Reply:
Also, correct me if I am wrong, if you will set a default value for it, so it should be optional